scripts/zkai_attention_kv_stwo_fusion_mechanism_ablation_gate.py

```
@@ -0,0 +1,664 @@
+    outputs = [
+        (_assert_output_path(json_path, "json output path"), json.dumps(payload, indent=2, sort_keys=True) + "\n"),
+        (_assert_output_path(tsv_path, "tsv output path"), to_tsv(payload)),
+    ]
+    for path, text in outputs:
+        tmp: pathlib.Path | None = None
+        write_error: OSError | None = None
+        try:
+            path.parent.mkdir(parents=True, exist_ok=True)
+            _assert_no_repo_symlink_components(path.parent, "output path")
+            if path.is_symlink():
+                raise FusionMechanismAblationGateError("output path must not include symlink components")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)